src/go/k8s/pkg/resources/configmap.go

```diff
@@ -50,24 +63,28 @@ func NewConfigMap(
 	}
 }
 
+// Ensure will manage kubernetes v1.ConfigMap for redpanda.vectorized.io CR
 func (r *ConfigMapResource) Ensure(ctx context.Context) error {
 	var cfgm corev1.ConfigMap
 
-	err := r.Get(ctx, r.Key(), &cfgm)
+	err := r.Get(ctx, r.key(), &cfgm)
```

Comment:
Did you get a linter complain because Key() and Obj() aren't used outside of the package? I suspect eventually we'll need to export them, but I'm good with the change for now.

Reply:
yeah sometimes the linter is ... not helpful :D

Reply:
That was my decision to hide for now the function, because I was lazy to create proper comment for the `Key` exported function. I can bring back the exported function and propose the comment as follows:
> 	Key returns namespace/name object that is used to identify object.
	For reference please visit types.NamespacedName docs in k8s.io/apimachinery

Reply:
I would vote for bringing back and adding the comment if it doesn't delay you much.